website/docs/r/compute_backend_service.html.markdown

```diff
@@ -133,3 +133,11 @@ exported:
 * `fingerprint` - The fingerprint of the backend service.
 
 * `self_link` - The URI of the created resource.
+
+## Import
+
+Backend services can be imported using the `name`, e.g.
+
+```
+$ terraform import google_compute_backend_service.website my_backend_service
```

Comment:
Nit: Can this line also be updated to import the example at the top of the page? It's a pattern I am trying to start going forward with #119. That way, readers of the Import documentation will be able to see a concrete example of how to structure an import for a resource based on an existing .tf configuration.